pkg/kudoctl/cmd/plan/plan_history.go

```diff
@@ -56,18 +58,21 @@ func planHistory(options *Options, settings *env.Settings) error {
 	tree := treeprint.New()
 	timeLayout := "2006-01-02T15:04:05"
 
-	for _, p := range instance.Status.PlanStatus {
+	plans, _ := funk.Keys(instance.Status.PlanStatus).([]string)
+	sort.Strings(plans)
+
+	for _, p := range plans {
```

Comment:
```suggestion
	for _, p := range plans {
	  plan := instance.Status.PlanStatus[p]
```

What do you think about this, and then using `plan.LastUpdatedTimestamp` etc. instead of accessing the `instace.Status.PlanStatus[p]` everytime?

Reply:
Yup that's a lot more readable 👍